build/rpmbuild_internal.h

```diff
@@ -49,6 +50,18 @@ struct Source * next;
 
 typedef struct Package_s * Package;
 
+/* Do not use number 0 for a line type */
```

Comment:
Hmm, why not? Having that LINE_DEFAULT kind of thing to represent all the normal lines seemed useful. Perhaps you misunderstood what I meant by the earlier comments?